dns-hosting.md

```diff
@@ -0,0 +1,20 @@
+---
+title: DNS hosting
+layout: recommendation
+expires: 2018-01-01
+---
+To make services available on the internet they need to have DNS records to let people know where they are.
+service.gov.uk domains are delegated to [nameservers](https://www.gov.uk/service-manual/technology/get-a-domain-name#check-where-youll-host-your-dns) controlled by service teams.
+
+
+## Principles
+
+- DNS entries should be version controlled unless autogenerated
+- Nameservers should be redundant enough for your service.
+
+
+## Recommendations
+
+- use route53 from AWS
+- if you need more redundancy also use Google and or DYN
+- use [govuk-dns](https://github.com/alphagov/govuk-dns) to version control and publish your records if you need to
```

Comment:
this still needs some work IMO to make it more accessible to other teams, happy to refactor if there is a requirement